src/tutorial/cli-args-struct.rs

```diff
@@ -6,12 +6,11 @@ struct Cli {
 }
 
 fn main() {
+    let pattern = std::env::args().nth(1).expect("no pattern given");
+    let path = std::env::args().nth(2).expect("no path given");
 
-let pattern = std::env::args().nth(1).expect("no pattern given");
```

Comment:
What's the reason for including `fn main() {` part in that snippet?

Reply:
I believe it is clearer and easier to read and understand when the context (here `main`) is given; especially for Rust newbies.

(Rather than letting the readers figure that out for themselves or explaining "the following line goes into the main function", etc.)

Reply:
Wouldn't that be covered by `, after fn main() {` part that was deleted in this PR?

Reply:
Sorry if I cannot express my point adequately.

Yes, the information is indeed given in the text. But I think, that

> Getting the raw arguments this way is quite easy (in file src/main.rs):
> ```rust
> fn main() {
>     let pattern = std::env::args().nth(1).expect("no pattern given");
>     let path = std::env::args().nth(2).expect("no path given");
> }
> ```

can be read more fluently and is easier to grasp than

> Getting the raw arguments this way is quite easy (in file src/main.rs, after fn main() {):
> ```rust
> let pattern = std::env::args().nth(1).expect("no pattern given");
> let path = std::env::args().nth(2).expect("no path given");
> ```

For instance, the first thing I stumbled over in this sentence was the strangely looking `{)` near the end.

I have just finished working through several chapters in the Rust book, and what I really liked there about the code examples is that they often are self contained, runnable, full examples. Since space is not really an issue in an online tutorial, I think that's generally a good thing.